Python/Product/Common/Strings.resx

```diff
@@ -495,6 +495,221 @@ The web.config file will be configured to resolve URIs matching this pattern to
+    <value>Go online and help me find a Python interpreter</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_FoundRequirements" xml:space="preserve">
+    <value>We found a requirements.txt file.</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_FoundRequirementsHelp" xml:space="preserve">
+    <value>This file contains external packages that are required by your project. We can use pip to download and install these dependencies automatically.</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_DownloadInstallPackages" xml:space="preserve">
+    <value>Download and _install packages</value>
+  </data>
+  <data name="AddVirtualEnvironmentWindow_Title" xml:space="preserve">
+    <value>Add Virtual Environment</value>
+  </data>
+  <data name="CannotFindConda" xml:space="preserve">
+    <value>Cannot find conda</value>
+  </data>
+  <data name="CurrentWorkingDirectoryCaption" xml:space="preserve">
+    <value>Working Directory ({0})</value>
+    <comment>{0} is a folder path</comment>
+  </data>
+  <data name="ProjectDoesNotSupportedMSBuild" xml:space="preserve">
+    <value>Project does not support MSBuild</value>
+  </data>
+  <data name="CustomCommandExecutingScript" xml:space="preserve">
+    <value>Executing {0} {1}</value>
+    <comment>{0} is filename
+{1} is arguments</comment>
+  </data>
+  <data name="CustomCommandExecutingModule" xml:space="preserve">
+    <value>Executing -m {0} {1}</value>
+    <comment>{0} is module name
+{1} is arguments</comment>
+  </data>
+  <data name="CustomCommandExecutingOther" xml:space="preserve">
+    <value>Executing {0} {1}</value>
+    <comment>{0} is filename
+{1} is arguments</comment>
+  </data>
+  <data name="FilePublisherIncorrectUsernameOrPassword" xml:space="preserve">
+    <value>Incorrect user name or password: {0}</value>
+    <comment>{0} is error details</comment>
+  </data>
+  <data name="FilePublisherDestinationDescription" xml:space="preserve">
+    <value>file path</value>
+  </data>
+  <data name="FtpPublisherDestinationDescription" xml:space="preserve">
+    <value>ftp server</value>
+  </data>
+  <data name="FtpPublisherDirExistsCheckException" xml:space="preserve">
+    <value>Failed to check if directory exists: {0}</value>
+    <comment>{0} is error details</comment>
+  </data>
+  <data name="FtpPublisherDirCreateException" xml:space="preserve">
+    <value>Failed to create directory: {0}</value>
+    <comment>{0} is error details</comment>
+  </data>
+  <data name="FtpPublisherUploadFileException" xml:space="preserve">
+    <value>Failed to upload file: {0}</value>
+    <comment>{0} is error details</comment>
+  </data>
+  <data name="PublishPropertyControl_LocationLabel" xml:space="preserve">
+    <value>Publishing folder location ({0}): </value>
+    <comment>{0} is a dynamically built list of location types
+Location types examples:
+ftp or file
+web, ftp or file
+azure, web, ftp or file</comment>
```

Comment:
We may need to precompute these lists if we can, but we'll see whether the localizers complain.